vllm/config/model.py

```diff
@@ -1040,6 +1040,31 @@ def verify_with_parallel_config(
         if parallel_config.enable_expert_parallel:
             self._verify_with_expert_parallelism()
 
+        # Compute num_redundant_experts if not specified
+        if parallel_config.eplb_config.num_redundant_experts is None:
+            if parallel_config.enable_eplb:
+                num_logical_experts = self.get_num_experts()
+                # EP size is TP * DP for EPLB
+                ep_size = (
+                    parallel_config.tensor_parallel_size
+                    * parallel_config.data_parallel_size
+                )
+                # Minimum value ensures at least 1 local physical expert per rank:
+                # (num_logical_experts + num_redundant_experts) / ep_size >= 1
+                min_redundant = max(0, ep_size - num_logical_experts)
```

Comment:
Would it make sense to set the `min_redundant` to a value closest divisible by ep_size. So that we would automatically could support unusual ep_sizes? Something like `(ep_size - num_logical_experts % ep_size) % ep_size`